### PR TITLE
scoring logic

### PR DESCRIPTION
**Pull request by meganweijiang**

### CODE REVIEW

File: src/components/data/questionsData.json [hunk 1]
```diff
@@ -1,94 +1,50 @@
...
-        "nextId": 3
-      }
-    ]
-    }
-  ]
\ No newline at end of file
+    "event": "Test",
+    "fact": "Testing"
+  }
+]
```
Comment (meganweijiang) [inline src/components/data/questionsData.json]: fake data for testing

Comment (lifeplayerone) [reply src/components/data/questionsData.json]: LGTM, I think there is also TransportationEffect right?

Comment (meganweijiang) [reply src/components/data/questionsData.json]: I think that's a typo and should be infrastructure 😆  Kelsey said there are only 2 variables

File: src/components/data/questionsData.json [hunk 2]
```diff
@@ -8,15 +8,13 @@
           "choiceId": 1,
           "option": "Choose and move to 2",
           "densityEffect": 1,
-          "infrastructureEffect": 1,
-          "nextId": 2
```
Comment (chethanagopinath) [inline src/components/data/questionsData.json]: thank you!

**Review by MGardo — APPROVED** [review]
Lgtm 🏅

**Review by chethanagopinath — APPROVED** [review]
lgtm!

**Pull request merged**